integrations/gradle-bloop/src/test/scala/bloop/integrations/gradle/ConfigGenerationSuite.scala

```diff
@@ -22,11 +22,11 @@ import bloop.engine.BuildLoader
 
 import scala.collection.JavaConverters._
 
-class ConfigGenerationSuite481 extends ConfigGenerationSuite{
+class ConfigGenerationSuite481 extends ConfigGenerationSuite {
```

Comment:
Looks like these sources were not formatted before and my commit formatted them instead